OverlayPlugin.Core/Controls/WSConfigPanel.cs

```diff
@@ -483,9 +498,20 @@ private void simpStartBtn_Click(object sender, EventArgs e)
                     p.StartInfo.RedirectStandardError = true;
                     p.StartInfo.RedirectStandardOutput = true;
 
+                    var ngrokTooOld = false;
+
                     DataReceivedEventHandler showLine = (_, ev) =>
                     {
-                        if (ev.Data != null) simpLogBox.AppendText(ev.Data.Replace("\n", "\r\n") + "\r\n");
+                        if (ev.Data != null)
+                        {
+                            simpLogBox.AppendText(ev.Data.Replace("\n", "\r\n") + "\r\n");
+                            // ERR_NGROK_120  Your ngrok agent version "<VERSION>" is no longer supported.
```

Comment:
A link to the current documentation for these error codes as a comment here would be appreciated as well.

Reply:
Added